src/main/resources/application.yaml

```diff
@@ -96,6 +96,9 @@ dcsa:
         onlyForEventClassifierCode:
           - EST
           - REQ
+        primaryReceivers:
```

Comment:
Include `only` in the name a la: `onlyWhenPrimaryReceiverIs` (so we know it is a filter)

Reply:
Done, thanks for the feedback :)